styling-and-themes/overview.md

```diff
@@ -13,7 +13,7 @@ Telerik UI for .NET MAUI comes with a built-in theme that controls the visual ap
 
 In addition to the Telerik theme, Telerik UI for .NET MAUI also provides a Platform theme with light and dark variants. The platform theme closely resembles the light and dark modes of the device that runs your MAUI application.
 
->The built-in Telerik UI for .NET MAUI theme and its swatches were introduced in version 8.0.0 (2024 Q4) and are currently in preview.
+>The built-in Telerik UI for .NET MAUI theme and its swatches were introduced in version 8.0.0 (2024 Q4) as a preview. With Telerik UI for MAUI 11.0.0 (2025 Q2) the Telerik theme and its swatches are official.
```

Comment:
Official is a tricky term. The industry usually recognizes this as General Availability (GA). You might also go the way of "fully supported/generally available/built-in feature/core feature".

Reply:
yes the theme is GA, now, it was a built-in feature with 8.0.0 version too, but with 11.0.0 it was officially released, it is still built-in. 
Fully supported is not appropriate, as there are controls that do not have theming.